shared/src/packages/drivers/libs/types/request/driver-get-all-by-business-request-dto.type.ts

```diff
@@ -0,0 +1,3 @@
+type DriverAllByBusinessRequestParameters = { businessId: number };
```

Comment:
this type is more appropriate to business package I think, and can be named like
type BusinessGetAllDriversRequestParameters